test/functoria/query/install.expected

```diff
@@ -0,0 +1,7 @@
+bin: [
+  "./main.exe" {"noop"}
+]
+etc: [
+  "./vote" {"vote"}
+  "./warn_error" {"warn_error"}
+]
```

Comment:
Where do these come from? I don't see any install stanzas generated

Reply:
the functoria tool is generating these (could be folded in dune-project later on I guess)